### PR TITLE
fix:  android build fail due to CREDENTIALS_API

### PR DESCRIPTION
**Pull request by WooWan**
Fixes build error
Build was failed, and i got following error:

`Note: Recompile with -Xlint:deprecation for details.
/Users/steavenbc/Documents/codes/ws-mobile/ws-mobile/node_modules/react-native-otp-verify/android/src/main/java/com/faizal/OtpVerify/OtpVerifyModule.java:54: error: cannot find symbol
.addApi(Auth.CREDENTIALS_API)
^
symbol: variable CREDENTIALS_API
location: class Auth
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
1 error

FAILURE: Build failed with an exception.

What went wrong:
Execution failed for task ':react-native-otp-verify:compileReleaseJavaWithJavac'.

fixes:  #108

### CODE REVIEW

**Review by KManiKumarReddy — APPROVED** [review]
NEED THIS FIX URGENTLY

**Comment by KManiKumarReddy** [comment]
URGENT: @faizalshap  please make this change and publish!!
Android build has been failing for everyone

**Comment by faizalshap** [comment]
I am on it

**Review by abhyudaysoni03 — APPROVED** [review]

**Review by royalbhati — COMMENTED** [review]
LGTM

**Comment by KManiKumarReddy** [comment]
@faizalshap , I could see deprecation warnings and the build failed even after making this change, please do verify
https://github.com/faizalshap/react-native-otp-verify/issues/108#issuecomment-1947776040

**Comment by shubhamdehaat** [comment]
@faizalshap - Any way to make this work in older versions? Since the older version of my app has a good installation base.

**Comment by faizalshap** [comment]
Which version?

**Comment by rahulkumar1409** [comment]
> Which version?

1.0.3

**Review by CodeWithMufaddal — COMMENTED** [review]
FIx this asap

**Review by CodeWithMufaddal — APPROVED** [review]
add this

**Comment by shubhamdehaat** [comment]
> Which version?

`1.1.4` but I think that version is working fine. Please fix issue in `1.1.6`

**Review by Saliheen — APPROVED** [review]
Working

**Comment by Piyush023** [comment]
Please Fix this issue asap

**Review by Piyush023 — APPROVED** [review]

**Review by rttss-sahil — APPROVED** [review]
seems fine

**Comment by IndrakumarR** [comment]
> Working

how ?

**Review by pierguinzani — APPROVED** [review]

**Comment by dravecky** [comment]
@royalbhati I think the fix is waiting for your approval

**Review by dravecky — APPROVED** [review]

**Comment by dravecky** [comment]
@faizalshap please approve and merge asap

**Comment by faizalshap** [comment]
Ok

**Comment by paveltar** [comment]
Any explanation about the reason why it happened?

**Comment by KManiKumarReddy** [comment]
> having same issue with 1.1.4

there's no way it will automatically work for previous versions, if you need to definitely use old versions you need to either patch it or if possible set playServicesVersion or googlePlayServicesVersion in your build.gradle. Refer to this comment for more details: https://github.com/faizalshap/react-native-otp-verify/issues/108#issuecomment-1947943784

**Comment by samcasmmm** [comment]
updated to 1.1.6. now working. thanks for reply

**Pull request merged**
